rentals.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "json_class": "Rental",
+    "data": {
+      "date": "5-6-22",
+      "book": {
+        "title": "New Book",
+        "author": "New Author"
+      },
+      "person": null
+    }
+  }
```

Comment:
- [ ] I have noticed that when data is saved into JSON file, the person does not belong to the rental created. Kindly ensure to create a relationship so that the person belongs to the rental. This will help to list all rentals for that particular person ☺️.

Reply:
Hi @AlphaNtihinduka 

Thanks for reviewing our code and advice. We will come up with a solution.